service/pipeline.ml

```diff
@@ -130,7 +131,7 @@ let build_with_cluster ~ocluster ~analysis ~master source =
         and+ build = Node.of_job `Built image ~label:"build"
         and+ tests = Node.of_job `Built tests ~label:"tests"
```

Comment:
the same thing is required for the tests I believe

edit: oops sorry for the double post

Reply:
Tests should be covered already by the `after:image` bit:
```ocaml
        let tests =
          let spec = test_spec ~platform pkg ~after:image in
          Build.v ocluster ~label:"test" ~base ~spec ~master source
```

Reply:
Oh I see! Sorry for the disturbance.